### ISSUE TITLE
Reward auto claimer

### ISSUE THREAD

**Issue by aVitomin** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#454]
Some streamers have rewards with day\week\month cooldown
I have enough points around 4 month but still can't get reward because some people take it faster

It will be cool if we have tools for set auto get reward when it will be available

**Comment by EMOEMOJAI** [comment]
In my opinion this doesn't get into the scope of the bot. This feature would just automate redeeming things and isn't really "farming points" in any way. The best it does is actually make you lose points. This is similar to #190 #196 in a way.

**Comment by Coldblackice** [comment]
Good idea. The bot already has a function to auto-claim drops, so I don't see why this wouldn't fit in that same vein.
